.github/workflows/nix.yml

```diff
@@ -83,7 +83,7 @@ jobs:
         with:
           configuration: ${{ matrix.configuration }}
       - name: upload archive
-        uses: actions/upload-artifact@v4
```

Comment:
These look like they've been downgraded

Reply:
Deliberately so. The upgrade was just yesterday and some runners are failing to run v4. It is making the CI unreliable.

Reply:
[heavy-wsnnk-kcbnl](https://github.com/XRPLF/rippled/actions/runs/8269704525/job/22626252882)
[heavy-wsnnk-rmmwk](https://github.com/XRPLF/rippled/actions/runs/8269704525/job/22626252585)
[heavy-wsnnk-2dfzj](https://github.com/XRPLF/rippled/actions/runs/8266936245/job/22623669891)

Failing runners are running version 2.300.2. Passing runners are running 2.314.1. Until we get every runner on version >= 2.314.1, I think it's best to downgrade the action.